draft-ietf-quic-transport.md

```diff
@@ -3014,11 +3016,11 @@ the 1-RTT packet number space will be increased by two.
 
 Each endpoint independently verifies and enables use of ECN by setting the IP
 header ECN codepoint to ECN Capable Transport (ECT) for the path from it to the
-other peer. Even if ECN is not used on the path to the peer, the endpoint MUST
-provide feedback about ECN markings received (if accessible).
+other peer. Even if not setting ECN codepoints on packets it transmits, the
+endpoint MAY provide feedback about ECN markings received (if accessible).
```

Comment:
SHOULD?

Reply:
SHOULD SGTM.

Also, can you remove the parens around if accessible?